app/services/pipeline_report_service.rb

```diff
@@ -274,6 +292,32 @@ def merge_contigs(contigs, counts_by_tax_level)
     end
   end
 
+  def merge_contigs_csv(contigs, counts_by_tax_level)
```

Comment:
Instead of adding a separate function that duplicates a lot from the original `merge_csv` function, would it be possible to just pass the boolean `csv` to the `merge_csv` function and do some `if csv` conditionals? This would avoid duplicating the code.